pyramid_openapi3/__init__.py

```diff
@@ -147,7 +149,8 @@ def explorer_view(request: Request) -> Response:
             settings = config.registry.settings
             if settings.get(apiname) is None:
                 raise ConfigurationError(
-                    "You need to call config.pyramid_openapi3_spec for explorer to work."
+                    "You need to call config.pyramid_openapi3_spec for the explorer "
+                    "to work."
```

Comment:
I'm guessing this change is to cut the line length from 86 chars to below 80. 

This project allows 10% leeway over the 80 chars limit. The rationale is that usually one does not get a speeding ticket for going 2 miles over the limit. More in this talk: https://www.youtube.com/watch?v=wf-BqAjZb8M

Reply:
Ahh, yeah it turns out my linter was not reading from the .flake8 file properly (and I have my line-length set to a strict 88), and this line was 89 characters long.

Although 89 is larger than 10% of 80 so I am unsure why `B950` wasn't complaining about it.

Reply:
Ah no, it's over 88 chars, sorry for the noise.